### PR TITLE
Erlang 21 Support

### PR DESCRIPTION
**Pull request by etxemag75**
It seems Erlang 21 removed the support to apply records to function names, created workaround that takes first element as module and pass the record to the Args.

### CODE REVIEW

**Comment by choptastic** [comment]
Thanks for submitting this. I appreciate sending something that works.  Unfortunately, there's a bit of a maintenance and readability nightmare here.  The whole `element(1, Bridge)` thing is a big problem.

The much simpler approach is to just use the `sbw` module. 

So `Bridge:query_params()` should just be `sbw:query_params(Bridge)`.

I'll go through and fix these and you'll still get credit for the work :).  But this definitely needs to be fixed :).

Thanks!

File: include/wf.hrl [hunk 1]
```diff
@@ -947,11 +948,6 @@
         function                :: atom() | text(),
         args=[]                 :: [text()]
     }).
--record(if_value, {?ACTION_BASE(action_if_value),
```
Comment (choptastic) [inline include/wf.hrl]: Why did you remove this element?

Comment (etxemag75) [reply include/wf.hrl]: My wf.hrl was not the latest ('action_if_value' was added in April I see now).

Please ignore this, should still be there
